pkgs/development/python-modules/shapely/default.nix

```
@@ -58,6 +51,15 @@ buildPythonPackage rec {
 
   disabledTests = [
     "test_collection"
+  ] ++ lib.optionals stdenv.isDarwin [
```

```suggestion
  ] ++ lib.optionals (stdenv.isDarwin && stdenv.isAarch64) [
```
These only fail on aarch64, on x86_64 they pass.